woocommerce/woocommerce-gutenberg-products-block
# Fix Misplacement of Shipping Radio Buttons

Fixes #10027

### Testing Instructions
1. Create any test site.
2. Install and activate all the required plugins
3. Add sample products.
4. Add some Shipping methods.
5. Create new test page.
6. Add Cart block.
7. Change two-column grid to stacked column using align option.
8. Observe that "Shipping" radio buttons are not misaligned.

### Screenshots



| Before | After |
| ------ | ----- |
| ![image](https://github.com/woocommerce/woocommerce-blocks/assets/6242098/a1ab4870-88cd-4f31-8dab-fb754bff55d6) | <img width="701" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5656702/1d7a62cf-14aa-4d27-bb21-04d9d597160b"> |

### Changelog

> Fix an issue where the radio buttons for shipping methods were incorrectly aligned on the Cart block in the post and site editor.
